assets/src/blocks/amp-story-text/edit.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { PanelBody, SelectControl } from '@wordpress/components';
+import { RichText, InspectorControls, FontSizePicker, withFontSizes } from '@wordpress/editor';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import getTagName from './get-tag-name';
+import { AMP_STORY_FONTS } from '../../helpers';
+
+function TextBlock( {
+	attributes,
+	setAttributes,
+	className,
+	fontSize,
+	setFontSize,
+} ) {
+	const { placeholder, content, type, ampFontFamily } = attributes;
+	const tagName = getTagName( attributes );
+
+	const fontSizeClass = fontSize.class || undefined;
+
+	return (
+		<Fragment>
+			<InspectorControls>
+				<PanelBody title={ __( 'Text Settings', 'amp' ) }>
+					<SelectControl
+						key={ 'font-family' }
+						label={ __( 'Font family', 'amp' ) }
+						value={ ampFontFamily }
+						options={ AMP_STORY_FONTS }
+						onChange={ function( value ) {
+							setAttributes( { ampFontFamily: value } );
+						} }
+					/>
+					<FontSizePicker
+						value={ fontSize.size }
+						onChange={ setFontSize }
+					/>
+					<SelectControl
+						label={ __( 'Select text type', 'amp' ) }
+						value={ type }
+						onChange={ ( selected ) => setAttributes( { type: selected } ) }
+						options={ [
+							{ value: 'auto', label: __( 'Automatic', 'amp' ) },
+							{ value: 'p', label: __( 'Paragraph', 'amp' ) },
+							{ value: 'h1', label: __( 'Heading 1', 'amp' ) },
+							{ value: 'h2', label: __( 'Heading 2', 'amp' ) },
```

Comment:
From UX perspective it could make sense to have the options in the block's toolbar as well, like in case of the heading block for example. Maybe not that important within this PR though. Thoughts?
cc @westonruter

Reply:
Good question. I suppose it depends on how frequently we expect users to be changing the setting manually. Fine to follow-up in another PR.